src/Symfony/Component/PropertyInfo/Extractor/ReflectionExtractor.php

```diff
@@ -209,6 +194,140 @@ public function isInitializable(string $class, string $property, array $context
         return false;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getReadAccessor(string $class, string $property, array $context = []): ?ReadAccessor
+    {
+        try {
+            $reflClass = new \ReflectionClass($class);
+        } catch (\ReflectionException $e) {
+            return null;
+        }
+
+        $allowGetterSetter = $context['enable_getter_setter_extraction'] ?? false;
```

Comment:
i'd prefer constants over magic strings for options

Reply:
I also want to use constant in the long term, not sure where to put them however :s